http-server/src/main/java/org/triplea/modules/user/account/login/LoginModule.java

```diff
@@ -108,7 +116,6 @@ private ApiKey recordAnonymousLoginAndGenerateApiKey(
     return recordLoginAndGenerateApiKey(loginRequest, systemId, playerChatId, ip, false);
   }
 
-  // TODO: Project#12 also update access log
   @SuppressWarnings("unused")
```

Comment:
Is this method still unused?

Reply:
Looks like no, not sure how long the suppression was out of date for, looks like perhaps for some time.